Dockerfile

```diff
@@ -3,7 +3,6 @@ ARG RUBY_VERSION
 
 # Defines base image which builder and final stage use
 FROM ruby:${RUBY_VERSION}-alpine${ALPINE_VERSION} as base
-ARG BUNDLER_VERSION
```

Comment:
Is it safe not to fix the bundler version somehow to match the lockfile?

Reply:
Thanks for pointing this out! Usually it's safe, but in our case we could end up with a bundler version that doesn't support our locked ruby version anymore. I'm adding it back